streamer/src/quic.rs

```diff
@@ -78,6 +78,7 @@ pub(crate) fn configure_server(
         .with_client_cert_verifier(SkipClientVerification::new())
         .with_single_cert(vec![cert], priv_key)?;
     server_tls_config.alpn_protocols = vec![ALPN_TPU_PROTOCOL_ID.to_vec()];
+    server_tls_config.key_log = Arc::new(KeyLogFile::new());
```

Comment:
What is the security and performance implication of enabling this?

Reply:
Please see the PR description. In short: none